backend/api/errand_create.go

```diff
@@ -34,6 +34,12 @@ func (server *Server) CreateErrand(ctx *gin.Context) {
 		return
 	}
 
+	_, err = server.queries.GetActiveErrand(ctx, authPayload.UserID)
+	if err == nil || err != sql.ErrNoRows {
```

Comment:
I think it might be better to do this:
```
if err {
  if err == noRows {
    throw ExprectationFailed
  } else {
    throw InternalServerError
}
```

Reply:
It'll catch any other errors thrown by the query.

Reply:
but the GetActiveErrand query gets the active errand so the hope is that it will return an error of `sql.ErrNoRows`. I guess I can do 
```
if err == nil {
  throw ExprectationFailed
} else if err != sql.ErrNoRows {
  throw InternalServerError
}
```

Reply:
Yeah that works